packages/docs/docs/getting-started/layouts.mdx

```diff
@@ -107,6 +107,65 @@ useful properties are listed below:
 <ApiSnippet url={'/api/2d/components/Layout#justifyContent'} />
 <hr />
 
+### Disabling `layout` on a child node
+
+Sometimes it can be useful to remove the node from the node tree. This way the
```

Comment:
> remove the node from the node tree

That's not what disabling `layout` does. The node will still be the child of its parent, even when `layout` is disabled.